jenkins/e2e-image/e2e-runner.sh

```diff
@@ -84,6 +84,54 @@ function running_in_docker() {
     grep -q docker /proc/self/cgroup
 }
 
+### FUNCTIONS TO SUPPORT get-kube.sh ###
+
+# Moves local tarballs into a local path suitable for get-kube.sh, and sets
+# KUBERNETES_RELEASE and KUBERNETES_RELEASE_URL accordingly.
+function set_release_vars_from_local_gcs_stage() {
```

Comment:
I get that, and my goal is to simultaneously a) unblock your pulling binaries from the tarball and b) either ensure that the complexity of this file decreases in the process or else the additional complexity is tested.

However if my opinion is an outlier on the team and the rest of our rotation is eager to support this I will be happy to defer to the consensus here.